src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let getCallCounter = 0;
+
+  return fistValue => secondValue => {
```

Comment:
don't create functions like this, this code is hard to read
```suggestion
  return (fistValue) => {
   return (secondValue) => {...} 
  }
```